packages/perseus/src/widgets/interactive-graphs/graphs/point.tsx

```diff
@@ -52,7 +52,12 @@ function PointGraph(props: Props) {
         if (focusedIndex != null) {
             pointsRef.current[focusedIndex]?.focus();
         }
-    }, [props.graphState.focusedPointIndex, pointsRef]);
+    }, [
+        props.graphState.focusedPointIndex,
+        // Ensure that we re-focus if a point is deleted.
+        props.graphState.coords.length,
```

Comment:
While testing, this seemed to work well for the bug we were facing without causing any additional refreshes for other interactions. 

It seemed like a better solution than refactoring all of the Polygon and Point graph types/logic, but I'm open to discussion on it!